erc20_moc/Bet.noblock.js

```diff
@@ -234,21 +236,20 @@ class Bet {
       }
       // TODO: implement a method to transfer several tokens in one call (like batch transfer)
       // get the collateral from provides
-      await this.collateralToken.transferChain(dbClient, provider, this.walletId, amount);
+      await this.collateralToken.transferChain(dbClient, provider, this.walletId, 'eth', 'bet', amount);
```

Comment:
why are we using `eth` namespace? the WFAIR for the liqudiity as WFAIR for betting should be in `usr`. @SamirHodzic please verify